### PR TITLE
fix: mind map issues

### PR DESCRIPTION
**Pull request by doouding**
Fixes [BS-1054](https://linear.app/affine-design/issue/BS-1054/mind-map-拖拽同级节点调整顺序的有效触发区域太小)
Fixes [BS-1317](https://linear.app/affine-design/issue/BS-1317/mind-map-编辑中，undo-需要两次才响应)
Fixes [BS-1284](https://linear.app/affine-design/issue/BS-1284/mind-map-软换行之后，节点没有立刻增加新的行空间)
Fixes [PD-1660](https://linear.app/affine-design/issue/PD-1660/radial-雷达视图在拖拽之后，左右节点不平衡分布了)
Fixes [BS-1334](https://linear.app/affine-design/issue/BS-1334/增加-mind-map-block-级别导入埋点)
Fixes [PD-1214](https://linear.app/affine-design/issue/PD-1214/拖拽非根节点，建议去掉对齐效果)
Fixes [BS-1316](https://linear.app/affine-design/issue/BS-1316/选中节点的时候，快捷键会误触发编辑)

### CODE REVIEW

File: packages/affine/block-surface/src/element-model/mindmap.ts [hunk 1]
```diff
@@ -168,34 +168,11 @@ export class MindmapElementModel extends GfxGroupLikeElementModel<MindmapElement
...
-    ) {
-      const mid = Math.ceil(tree.children.length / 2);
-
-      tree.right.push(...tree.children.slice(0, mid));
-      tree.left.push(...tree.children.slice(mid));
-    } else {
-      tree.right.push(...tree.children.slice(0, splitPoint + 1));
-      tree.left.push(...tree.children.slice(splitPoint + 1));
-    }
+    const splitPoint = Math.ceil(tree.children.length / 2);
```
Comment (Saul-Mirone) [inline packages/affine/block-surface/src/element-model/mindmap.ts:172]: Not really related to this pr, but is it possible now to move this model to `affine-model`?

File: packages/affine/model/src/consts/text.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ export const FontFamilyList = Object.entries(FontFamilyMap) as {
 }[FontFamily][];
 
 export enum TextResizing {
-  AUTO_WIDTH,
+  AUTO_WIDTH_AND_HEIGHT,
   AUTO_HEIGHT,
```
[the remark this answers is not in the document]
Comment (doouding) [reply packages/affine/model/src/consts/text.ts:67]: The behavior is different. AUTO_HEIGHT only increases the height; the width is fixed.

File: packages/affine/block-surface/src/element-model/mindmap.ts (hunk 1 above)
Comment (doouding) [reply packages/affine/block-surface/src/element-model/mindmap.ts:172]: @doodlewind seems to be working on this.

**Review by Saul-Mirone — APPROVED** [review]
LGTM

**Pull request merged**